### PR TITLE
[stable/msoms] support custom tolerations

### PR DESCRIPTION
**Pull request by derekperkins**
I have taints in my cluster, but I still want the OMS DaemonSet to run on all nodes. This allows for custom tolerations to be added, similar to the custom nodeSelector option.

cc @keikhara

### CODE REVIEW

File: stable/msoms/values.yaml [hunk 1]
```diff
@@ -32,3 +32,7 @@ resources:
 # Node selector - default to Linux as cannot deploy OMS agent container to Windows nodes
 nodeSelector:
   beta.kubernetes.io/os: linux
+
+# optional tolerations to handle custom taints
+# tolerations:
```
Comment (unguiculus) [inline stable/msoms/values.yaml]: Create an empty list as default:

`tolerations: [] `

File: stable/msoms/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: msoms
-version: 0.1.3
+version: 0.1.4
```
Comment (unguiculus) [inline stable/msoms/Chart.yaml]: Bump the minor version.

**Comment by derekperkins** [comment]
Thanks for your help @unguiculus!

**Pull request merged**